blazor/datagrid/virtualization.md

```diff
@@ -167,6 +167,8 @@ N> Column's [Width](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grid
 
 N> The collapsed/expanded state will persist only for local dataSource while scrolling.
 
+N> When trying to AutoFit the columns in the Grid with column virtualization, AutoFit/ AutoFitAll width is applied only for the columns which are in the view port.
```

Comment:
N> When trying to AutoFit the columns in the Grid with column virtualization, AutoFit/AutoFitAll width is applied only for the columns in the viewport.